atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -132,20 +132,36 @@ def add_aliases(self, base_labels, df_labels, plugin_labels):
         applied_alias = False
         not_applied = []
 
-        for old, new in self.aliases.items():
-            if old in all_labels:
-                if new in new_labels:
-                    if all_labels[old] != all_labels[new]:
```

Comment:
Was it not any good to just set `all_labels[old] = all_labels[new]` at this point?